ocs_ci/ocs/perftests.py

```diff
@@ -70,6 +79,91 @@ def teardown(self):
         if hasattr(self, "operator"):
             self.operator.cleanup()
 
+        now_data = self.get_cephfs_data()
+        # Wait 1 minutes for the backend deletion actually start.
+        log.info("Waiting for Ceph to finish cleaning up")
+        time.sleep(60)
+
+        # Quarry the storage usage every 2 Min. if no difference between two
+        # samples, the backend cleanup is done.
+        still_going_down = True
+        while still_going_down:
+            new_data = self.get_cephfs_data()
+            # no deletion operation is in progress
+            if abs(now_data - new_data) < 1:
+                still_going_down = False
+                # up to 2% inflation of usage is acceptable
+                if new_data > (self.used_capacity * 1.02):
+                    log.warning(
+                        f"usage capacity after the test ({new_data:.2f} GiB) "
+                        f"is more then in the begining of it ({self.used_capacity:.2f} GiB)"
+                    )
+            else:
+                log.info(f"Last usage : {now_data}, Current usage {new_data}")
+                now_data = new_data
+                log.info("Waiting for Ceph to finish cleaning up")
+                time.sleep(120)
+                still_going_down = True
+        log.info("Storage usage was cleandup")
+
+    def create_new_pool(self, pool_name):
+        """
+        Creating new Storage pool for RBD / CephFS to use in a test so it can be
+        deleted in the end of the test for fast cleanup
+
+        Args:
+            pool_name (str):  the name of the pool to create
+
+        """
+        if self.interface == constants.CEPHBLOCKPOOL:
+            self.ceph_cluster.create_new_blockpool(pool_name=pool_name)
+        elif self.interface == constants.CEPHFILESYSTEM:
+            self.ceph_cluster.create_new_filesystem(fs_name=pool_name)
+
+        self.ceph_cluster.set_pgs(poolname=pool_name, pgs=128)
+        self.ceph_cluster.set_target_ratio(
+            poolname="ocs-storagecluster-cephblockpool", ratio=0.24
+        )
+        self.ceph_cluster.set_target_ratio(
+            poolname="ocs-storagecluster-cephfilesystem-data0", ratio=0.24
+        )
```

Comment:
Here are we modifying for existing default pools so that we don't see PG issues?

Reply:
part of it yes. 
since the default pool target_size_ratio is 0.49 the new pool is created with target_size_ratio of 0.49, and if we do not change the default pools to lower it, the actual_size_ratio will be 0.33 and not 0.5, so we reduce the ratio of the default pools so the new pool will have actual size ratio of 0.5.
in the end of the test we will reset is back to 0.49